sysadmin/build.yml

```diff
@@ -0,0 +1,617 @@
+        mode: 0770
+      when: not install_solr_data
+
+    - name: copy over schema.xml
+      copy:
+        src: "../solr-config/Solr-7/schema.xml"
+        dest: /var/solr/data/open-context/conf/schema.xml
+        owner: solr
+        group: solr
+        mode: 0770
+      when: not install_solr_data
+
+    # download the solr backup 
+    - name: download solr backup 
+      get_url:
+        url: "{{GROUP_SECRETS.SOLR_BACKUP_URL}}"
+        dest: "/tmp/{{GROUP_SECRETS.SOLR_BACKUP}}"
+        mode: 0640
+        owner: solr
+        group: solr
+      when: install_solr_data
+
+    - name: create /tmp/solr_backup
+      file: path="/tmp/solr_backup" state=directory owner=solr group=solr mode=0770
+      become: yes
+      when: install_solr_data
+
+    # unarchive the solr zip file
+    - name: unzip solr zip file
+      unarchive:
+        src: "/tmp/{{GROUP_SECRETS.SOLR_BACKUP}}"
+        dest: /tmp/solr_backup
+        remote_src: true
+        owner: solr
+        group: solr
+      when: install_solr_data
+
+    # # this didn't work -- time out on some privilege escalation
+    # - name: create open-context core 
+    #   command: "/opt/solr/bin/solr create_core -c open-context"
+    #   become: true
+    #   become_method: su
+    #   become_user: solr
+    #   become_flags: '-s /bin/bash'    
+
+    # mv /tmp/solr_backup/oc-solr-6/server/solr/open-context/* /var/solr/data/open-context/
+
+    - name: stat solr backup data 
+      stat: path=/tmp/solr_backup/oc-solr-6/server/solr/open-context
+      register: oc_stat
+      become: yes
+      when: install_solr_data
+
+    # TO DO -- parameterize oc-solr-6
+    - name: Move solr backup to solr data directory
+      command: sudo -u solr bash -c "rm -rf /var/solr/data/open-context/; mv -f /tmp/solr_backup/oc-solr-6/server/solr/open-context /var/solr/data/"
+      # when: oc_stat.exists and install_solr_data
+      when: install_solr_data
+
+    # issue command to reload core
+    - name: issue command to reload core
+      command: curl "http://localhost:8983/solr/admin/cores?action=RELOAD&core=open-context&&wt=json"
+      when: install_solr_data
+
+    # then need a reload https://serverfault.com/questions/703031/how-do-i-add-a-solr-core-without-restarting-the-solr-server
+    # curl "http://localhost:8983/solr/admin/cores?action=RELOAD&core=open-context&&wt=json"
+    # curl "http://octest.raymondyee.net:8983/solr/admin/cores?action=RELOAD&core=collection1&wt=json"
+
+    # - name: end play
+    #   meta: end_play
+ 
+    # - name: grab ssh config ip
+    #   debug:
+    #      var: "hostvars[inventory_hostname]['ansible_env'].SSH_CONNECTION.split(' ')[2]"
+
+    # - name: Display all variables/facts known for a host
+    #   debug:
+    #     var: hostvars[inventory_hostname]
+    #   tags: debug_info
+
+    # - name: Getting PYTHONPATH
+    #   local_action: shell python -c 'import sys; print(":".join(sys.path))'
+    #   register: pythonpath
+
+    # - debug:
```

Comment:
Another thing that I've not built into build.yml is the series of database fixes (fixtures?) that might need to be run on old data.  I've documented them at [Some data fixes to apply to Open Context data](https://gist.github.com/rdhyee/f92805dd45de5a4b799963d50154d6e8)